app/mobile/BuPazar/app/src/main/java/com/example/bupazar/model/NotificationResponse.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.bupazar.model
+
+import com.google.gson.annotations.SerializedName
+import java.io.Serializable
+
+
+data class NotificationResponse(
+    @SerializedName("id") val id: Int?,
+    @SerializedName("text") val notificationText: String,
```

Comment:
why some of them has '?' but other do not have, we should follow strict convention.